components/brave_wallet_ui/panel/reducers/panel_reducer.ts

```diff
@@ -190,4 +194,14 @@ export const createPanelReducer = (initialState: PanelState) => {
 
 const reducer = createPanelReducer(defaultState)
 export const panelReducer = reducer
+export const persistedPanelReducer = persistVersionedReducer(panelReducer, {
+  key: 'panel',
+  version: PERSISTED_STATE_VERSION,
+  blacklist: [
```

Comment:
Would it make sense to update this one to an allow list too? What possible panels would be able to be added to this at the moment?

Reply:
List is available here: https://github.com/brave/brave-core/blob/feat-persist-wallet-redux-store/components/brave_wallet_ui/constants/types.ts#L279

Reply:
Now using a whitelist